src/nfv_test_api/config.py

```diff
@@ -39,4 +41,13 @@ def get_config(config_file: Optional[str] = None, config_dict: Optional[Dict[str
             config_dict = yaml.safe_load(stream)
 
     CONFIG = Config(**config_dict)
+        
+    gnb_config_folder = pathlib.Path(CONFIG.gnb_config_folder)
+    if not gnb_config_folder.exists():
+        gnb_config_folder.mkdir()
```

Comment:
```suggestion
    gnb_config_folder.mkdir(parents=True, exists_ok=True)
```
You can simplify this like this